_projects/enhance-and-improve-coala-community-and-projects-website.md

```diff
@@ -0,0 +1,160 @@
+
+    b. Add information about new endpoints at coala-webserivces homepage.
+
+    c. Add [contributor's Geolocation](https://github.com/coala/community/issues/18)
+
+    d. Add [Organisation Teams](https://github.com/coala/community/issues/12)
+
+    e. Add [Mentors](https://github.com/coala/community/issues/11)
+
+    f. Add **Newcomer Invitation** [form](https://github.com/coala/community/issues/89#issuecomment-468955279)
+
+    g. Add **Promotion to Developer** [form](https://github.com/coala/community/issues/89)
+
+    h. Manage live data of community members
+
+    i. Addition of more data fields in **Users** db model
+
+    j. Add **Login and Sign-up** using GitHub
+3. Make Improvements (or fix bugs) in coala [community website](https://community.coala.io/)
+
+    a. Build failure for more than one remote [#235](https://github.com/coala/community/issues/235)
+
+    b. Netlify build failure for fork [#232](https://github.com/coala/community/issues/232)
+
+#### CODING PHASE 2
+
+1. Implementation of listed enhancements in coala [community website](http://community.coala.io/)
+
+    a. Create **Profile Page**
+
+    b. Leaderboard on **[Homepage](https://coala-community.netlify.com/)**
+
+    c. Enhancement in **[Gamification Leaderboard](https://coala-community.netlify.com/gamification/)**
+
+2. Implementation of listed enhancements in [projects website](https://projects.coala.io)
+
+    a. Add **[Filters](https://github.com/coala/projects/issues/559)** for searching projects
+
+3. Make Improvements in coala [community website](https://community.coala.io/)
+
+    a. **[Gamification Leaderboard](https://coala-community.netlify.com/gamification/)**
+
+    b. **[UI](https://coala-community.netlify.com/)**
+      * [Re-designing of the website](https://github.com/coala/community/issues/44) and making it responsive.
+      * [Inactive Issue](https://coala-community.netlify.com/static/inactive-issues.json) and [unassigned issues activity](https://coala-community.netlify.com/static/unassigned-issues.json) web pages shows
+        just JSON response, it should be displayed in proper format with some
+        extra important information.
+      * [Meta-Review data](https://coala-community.netlify.com/meta-review/) web pages should be redesigned to make it look good.
+
+#### CODING PHASE 3
+
+1. Implementation of listed enhancements in [projects website](https://projects.coala.io)
+
+    a. [Improve](https://github.com/coala/projects/issues/394) the projects website for use by other GSoC orgs
+
+    b. Create [tab for all the google forms](https://github.com/coala/projects/issues/284)
+
+    c. Add [posts / calendar app](https://github.com/coala/projects/issues/560) to projects website
+
+    d. Create a tab for **GSoC Students**
+
+2. Implementation of listed enhancements in coala [community website](http://community.coala.io/)
+
+    a. Create a feedback form
```

Comment:
If you want to implement a Django form then it couldn't be done in the community website, as its a static website and we are not storing any data anywhere permanently.

Reply:
Won't it be good if we are able to store data on coala community website permanently? I guess this is achievable by removing the `removal of db.sqlite` cli instruction from the bash script.
Although, the data for `feedback form` can even be stored in `coala-webservices database`.

Reply:
>..achievable by removing the removal of db.sqlite cli instruction from the bash script.

No this is a static website, it does not matter if you have db.sqlite file or not the whole data will be updated with the new api data when the app is deployed. And iirc all the apps in the community website are built following the same principle.

Reply:
>Although, the data for feedback form can even be stored in coala-webservices database.

Sounds cool.

Reply:
> it does not matter if you have db.sqlite file or not the whole data will be updated with the new api data when the app is deployed

Yeah, I just checked pull requests and analyzed what's happening in the background. Removing `db.sqlite` instruction won't make any changes 😢 👍